### PR TITLE
Porfolio: works, about me and summary structure and style

### PR DESCRIPTION
**Pull request by chimake**
Added colour variables, style and aligned works, about me and summary section. Also, set up the structure for works, about me and summary section (changes a few ids to class, and create cards, and lists). Finally adding media files.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -57,39 +68,276 @@ a#img_link {
...
 
 .icon_image {
   padding: 10px;
 }
+
+#works {
+  height: 100%;
+  background-color: var(--white-shade-1);
+  padding: 16px;
+  display: grid;
```
Comment (joskalenda) [inline assets/css/style.css:116]: ❤️

File: assets/css/style.css [hunk 2]
```diff
@@ -57,39 +68,276 @@ a#img_link {
...
+  gap: 10px;
+  grid-template-rows: 100vh 100vh 100vh 100vh;
+  justify-content: space-around;
+}
+
+.card {
+  height: 98vh;
+  display: flex;
```
Comment (joskalenda) [inline assets/css/style.css:124]: ❤️

File: index.html [hunk 3]
```diff
@@ -54,7 +54,170 @@ <h2 id="name_title">I’m Chima Glad to see you!</h2>
                     </ul>
                 </nav>
             </div>                        
-        </section>        
+        </section>      
+        <section id="works">
+            <div class="card">
```
Comment (joskalenda) [inline index.html:59]: - [ ] I have really appreciated the way you wrote your HTML code, so clear and follows the requirement, but I have some issues to raise concerning the design



- [ ]  Let start by the space between the headline and the first card, seeing the Figma image on the left, yours is smaller, please fix that up
- [ ] The background of your card section is darker, take a look at the image on the left and bring the necessary changes
- [ ] Your card width is to modify, consider adding it to match the original card design width

- [ ] Your card title's font-weight is to be touched a bit, make the necessary changes to have the same bold style as per the image on the left.

File: index.html [hunk 4]
```diff
@@ -54,7 +54,170 @@ <h2 id="name_title">I’m Chima Glad to see you!</h2>
...
+                <div class="project_description">
+                    <div class="project_title">Tonic</div>
+                    <div class="project_summary">
+                        <ul class="summary_list">
+                            <li class="summary_list_item no_disc">canopy</li>
+                            <li class="summary_list_item"> back end dev</li>
+                            <li class="summary_list_item">2015</li>
+                        </ul>
+                    </div>
+                    <p class="project_text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```
Comment (joskalenda) [inline index.html:72]: - [ ] Not much to say, all is clear here, your technologies color is not the same as the one on the left, please change that
- [ ] The space after your `see project` button is considerably wide 😄 , what do you think if you make a few modifications?
- [ ] After that you will have to update the space between your cards though, as highlighted in red,

File: index.html [hunk 5]
```diff
@@ -54,7 +54,170 @@ <h2 id="name_title">I’m Chima Glad to see you!</h2>
...
-        </section>        
+        </section>      
+        <section id="works">
+            <div class="card">
+                <a href="#" class="project_cta_link">
+                    <img src="assets/images/SnapshootPortfolio.png" alt="Project Name" class="snapshot_portfolio">
+                </a>
+                <div class="project_description">
+                    <div class="project_title">Tonic</div>
+                    <div class="project_summary">
```
Comment (joskalenda) [inline index.html:65]: - [ ] The project title (e.g., "Tonic") uses a header tag hx, seeing yours, you are wrapping it inside a div element, please change it.

File: assets/css/style.css (hunk 2 above)
Comment (chimake) [reply assets/css/style.css:124]: Thank you so much for your review. Just to confirm can I go ahead and merge?

File: index.html [hunk 6]
```diff
@@ -54,7 +54,170 @@ <h2 id="name_title">I’m Chima Glad to see you!</h2>
...
+                            <span class="accordio_cta">Languages
+                                <img src="assets/images/Union.png" alt="Collapse">
+                            </span>
+                            <ul class="breakdown_list_children">
+                                <li class="breakdown_list_children_item"><img src="assets/images/js.png" alt="Javascript"></li>
+                                <li class="breakdown_list_children_item"><img src="assets/images/html.png" alt="Html"></li>
+                                <li class="breakdown_list_children_item"><img src="assets/images/css.png" alt="Javascript"></li>
+                            </ul>
+                        </li>
+                        <li class="breakdown_list_item"><span class="accordio_cta">Frameworks <img src="assets/images/chevron.png" alt="Expand"></span></li>
```
Comment (joskalenda) [inline index.html:214]: - [ ] The image said it all, kindly add the correct spaces after the `get my resume` button
- [ ] And adjust the icons as pointed, take a look at the original design as reference

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @chimake , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Green linters
- [ ].. ✅ Good commit messages
- [ ].. ✅ Good readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NduatiKagiri — APPROVED** [review]
Hi @chimake ,

# APPROVED 💯  

Good job implementing the requested changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

ea0cb5a48e3337523ffe8d60f177879d

## Highlights ✔️ 
✅ GitHub Flow observed
✅ No Linters errors

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by citec-47 — COMMENTED** [review]
#From our review we conclude that the code base is up to date and there are no visible errors

**Review by ouasamine — COMMENTED** [review]
Hi @chimake 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Nice work on the UI 👍🏽 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**